src/Xamarin.Android.Build.Tasks/Microsoft.NET.Sdk.Android/WorkloadManifest.in.json

```diff
@@ -7,24 +7,22 @@
         "Microsoft.Android.Sdk",
         "Microsoft.Android.Ref.31",
         "Microsoft.Android.Ref.32",
+        "Microsoft.Android.Ref.33",
         "Microsoft.Android.Runtime.32.android-arm",
         "Microsoft.Android.Runtime.32.android-arm64",
         "Microsoft.Android.Runtime.32.android-x86",
         "Microsoft.Android.Runtime.32.android-x64",
```

Comment:
Can we also remove these, is there a case where they get used? They can also come down from NuGet.org if they are not in the workload.

Reply:
Good point, I can't think of anywhere these would be used anymore.